src/FreeAssAlgebra.jl

```diff
@@ -67,7 +67,7 @@ end
 
 function expressify(a::FreeAssAlgebra; context = nothing)
    return Expr(:sequence, Expr(:text, "Free associative algebra over "),
-                          expressify(base_ring(a)),
+                          string(base_ring(a)),
```

Comment:
This was relying on the fallback `expressify`. Does this change seem reasonable to you, @tthsqe12 ?

Reply:
This does not seems like a reasonable change. If anything, it should be
`expressify(base_ring(a), context = context)`. If expressify on the base ring broken or going to be broken soon?

Reply:
Isn't `string` falling back to the same thing eventually?

Reply:
This pr was canning the string fallback, to which I object.

Reply:
We are not canning the string fallback.

Reply:
Is this an agreement or a complaint?